src/cache/mod.rs

```diff
@@ -0,0 +1,25 @@
+// Make module "requests" private and re-export its contents so users can
+// import using "momento::cache::{Get, Set, ...}"
+mod requests;
+
+pub use requests::MomentoRequest;
+
+pub use requests::create_cache::{CreateCache, CreateCacheRequest};
+pub use requests::delete_cache::{DeleteCache, DeleteCacheRequest};
+pub use requests::flush_cache::{FlushCache, FlushCacheRequest};
+pub use requests::list_caches::{ListCaches, ListCachesRequest};
+
+pub use requests::scalar::get::{Get, GetRequest, GetValue};
+pub use requests::scalar::set::{Set, SetRequest};
+
+pub use requests::set::set_add_elements::{SetAddElements, SetAddElementsRequest};
+
+pub use requests::sorted_set::sorted_set_fetch_by_rank::{SortOrder, SortedSetFetchByRankRequest};
```

Comment:
if this SortOrder is really specific to SortedSets we might need to disambiguate that in some way, like name it SortedSetSortOrder or SortedSet::SortOrder or something. If it's not specific to sorted sets then we could move its definition to a more generic location. Just a nit; this is not really something specific to this PR, can sort it out later (pun intended)

Reply:
Looks like it's called SortedSetOrder in the other SDKs so will follow suit here